win32ss/user/ntuser/winpos.c

```diff
@@ -2103,10 +2103,7 @@ co_WinPosSetWindowPos(
                    IntInvalidateWindows( Parent, DirtyRgn, RDW_ERASE | RDW_INVALIDATE);
                    co_IntPaintWindows(Parent, RDW_NOCHILDREN, FALSE);
                 }
-                else
-                {
-                   IntInvalidateWindows( Window, DirtyRgn, RDW_ERASE | RDW_FRAME | RDW_INVALIDATE | RDW_ALLCHILDREN);
-                }
+                IntInvalidateWindows(Window, DirtyRgn, RDW_ERASE | RDW_FRAME | RDW_INVALIDATE | RDW_ALLCHILDREN);
```

Comment:
It is effectively doing two invalidates for WS_CHILD windows now?

Reply:
oh nvm, I can't read.

Reply:
The first one is for parent invalidation, and the second one is for the window itself

Reply:
Indeed, saw that a bit too late, hence why I resolved my comment :)